image/Dockerfile.laboratory

```diff
@@ -0,0 +1,14 @@
+FROM node:9.11-alpine
+
+WORKDIR /opt
+
+RUN apk add -qU --no-cache -t .devstuff git \
```

Comment:
`devstuff` is noob-speak. people usually use `fetch-deps` or `build-deps` (which is our case imo)